jQuery/bs-config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "port": 3000,
```

Comment:
same with React. Maybe change to 3030 or smth?

Reply:
are you planning to run React and jQuery at the same time?) I think a person usually debugs one project per time